### DISCUSSION TITLE
EXT_mesh_gpu_instancing support

### DISCUSSION THREAD

**Discussion by ibober** [google/model-viewer#4389] [Ideas]
Maybe it's out of most use cases, but it would be nice to support this extension to improve performance on huge models.

**Comment by elalish** [comment]
Our glTF support comes through three.js, so if they support it, so do we. It looks like they do: https://threejs.org/docs/index.html?q=gltf#examples/en/loaders/GLTFLoader, so I think you're in luck.

**Comment by ibober** [comment]
Oh, indeed, I didn't notice.
But could you help then to understand why I don't see this [model](https://drive.google.com/file/d/1TCJ0fRLDAV1oclskfa1Iau396vKn8cP1/view?usp=drive_link) in the [viewer](https://modelviewer.dev/editor/).

**Reply by ibober** [reply]
*it works in [threejs editor](https://threejs.org/editor/):
![image](https://github.com/google/model-viewer/assets/8260211/85876011-bbf3-4e99-bd38-6099c6de4127)

**Reply by elalish** [reply]
As best I can tell it is rendering. My hunch is that we're not calculating its bounding box correctly and so the camera isn't looking at it. If you know its dimensions/coordinates, can you try to set `camera-target` and `camera-orbit` to reasonable values for this model and see if it shows up? Then we'll know where the bug is.

**Reply by ibober** [reply]
Adding a mesh without "instancing" extension helped, thanks for pointing to the right direction.
